src/hotspot/share/classfile/classLoaderExt.cpp

```diff
@@ -227,7 +229,8 @@ void ClassLoaderExt::setup_search_paths(JavaThread* current) {
   ClassLoaderExt::setup_app_search_path(current);
 }
 
-void ClassLoaderExt::record_result(const s2 classpath_index, InstanceKlass* result) {
+void ClassLoaderExt::record_result(const s2 classpath_index, InstanceKlass* result,
+                                   const ClassFileStream* stream, bool redefined) {
```

Comment:
The `stream` argument is not used.

Reply:
Thanks for spotting this. I've fixed it.